lib/internal/process/task_queues.js

```diff
@@ -92,9 +92,6 @@ function processTicksAndRejections() {
 
 class TickObject {
   constructor(callback, args, triggerAsyncId) {
-    // This must be set to null first to avoid function tracking
-    // on the hidden class, revisit in V8 versions after 6.2
```

Comment:
Are these related to https://bugs.chromium.org/p/v8/issues/detail?id=5456 ? @apapirovski

Reply:
BTW, there does not seem to be a benchmark that passes different callbacks to `process.nextTick()` in different iterations (like a `for (...) process.nextTick(() => { ... closure })`) so the deopt storm probably can not even be manifested in the benchmarks.

Reply:
Yes but last I tried this, it was still an issue. We should create a benchmark that actually tests this if we want to be certain. (i recently tried this with timers and still saw perf penalty.)